packages/react/src/primitives/Select/Select.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Flex } from '../Flex';
+import { View } from '../View';
+import { IconExpandMore } from '../Icon';
+import { SelectProps } from '../types';
+import { ComponentClassNames } from '../shared';
+
+export const Select: React.FC<SelectProps> = (props) => {
+  const {
+    className,
+    size,
+    variation,
+    icon = <IconExpandMore size="large" />,
+    iconColor,
+    children,
+    placeholder,
+    isRequired,
+    ...rest
+  } = props;
+  return (
+    <View className="amplify-select-wrapper">
+      <View
+        as="select"
+        required={isRequired}
+        data-size={size}
+        data-variation={variation}
+        className={classNames(ComponentClassNames.Select, className)}
+        {...rest}
+      >
+        {placeholder && (
+          <option value="" disabled>
```

Comment:
I wonder if we should leave off the placeholder option and let customers pass `<option value="">Choose a State</option>` themselves. This would give them more control over what they would like the value of the placeholder to me (in case they wanted to do something like `<option value="DEFAULT">Customer placeholder prompt</option>`. I don't think it would be a big loss to not have a placeholder prop, and in fact make the component more flexible.

Reply:
Yea, that's a good idea. I mean the `placeholder` is optional. They can totally pass a custom one themselves.